PWGJE/PWGJE/AliAnalysisTaskJetCoreEmcal.cxx

```diff
@@ -0,0 +1,945 @@
+ * documentation strictly for non-commercial purposes is hereby granted   *
+ * without fee, provided that the above copyright notice appears in all   *
+ * copies and that both the copyright notice and this permission notice   *
+ * appear in the supporting documentation. The authors make no claims     *
+ * about the suitability of this software for any purpose. It is          *
+ * provided "as is" without express or implied warranty.                  *
+ **************************************************************************/
+
+#include <TClonesArray.h>
+#include <TList.h>
+#include "TChain.h"
+#include "TTree.h"
+#include "TMath.h"
+#include "TH1F.h"
+#include "TH2F.h"
+#include "TH3F.h"
+#include "THnSparse.h"
+#include "TCanvas.h"
+#include "TRandom3.h"
+#include "AliLog.h"
+
+#include <AliAnalysisManager.h>
+#include <AliVEventHandler.h>
+#include <AliVEvent.h>
+#include <AliVCluster.h>
+#include <AliVParticle.h>
+#include <AliLog.h>
+
+#include "AliTLorentzVector.h"
+#include "AliEmcalJet.h"
+#include "AliRhoParameter.h"
+#include "AliJetContainer.h"
+#include "AliParticleContainer.h"
+#include "AliClusterContainer.h"
+
+#include "AliAnalysisTaskJetCoreEmcal.h"
+
+/// \cond CLASSIMP
+ClassImp(AliAnalysisTaskJetCoreEmcal);
+/// \endcond
+
+/**
+ * Default constructor. Needed by ROOT I/O
+ */
+AliAnalysisTaskJetCoreEmcal::AliAnalysisTaskJetCoreEmcal() : 
+  AliAnalysisTaskEmcalJet(),
+  fHistManager(),
+	fCentMin(0.),
+	fCentMax(100.),
+	fTTLowRef(8.),
+	fTTUpRef(9.),
+	fTTLowSig(20.),
+	fTTUpSig(50.),
+	fNRPBins(50),
+	fFrac(0.8),
+	fJetEtaMin(-.5),
+	fJetEtaMax(.5),
+	fJetHadronDeltaPhi(0.6),
+	fJetContName(""),
+	fRandom(0),
+	fHistEvtSelection(0x0), 
+	fHJetSpec(0x0),
+	fh1TrigRef(0x0),
+	fh1TrigSig(0x0),
+	fh2Ntriggers(0x0),
+	fh2RPJetsC10(0x0),
+	fh2RPJetsC20(0x0),
+	fh2RPTC10(0x0),
+	fh2RPTC20(0x0)
+
+{
+}
+
+/**
+ * Standard constructor. Should be used by the user.
+ *
+ * @param[in] name Name of the task
+ */
+AliAnalysisTaskJetCoreEmcal::AliAnalysisTaskJetCoreEmcal(const char *name) : 
+  AliAnalysisTaskEmcalJet(name, kTRUE),
+  fHistManager(name),
+	fCentMin(0.),
+	fCentMax(100.),
+	fTTLowRef(8),
+	fTTUpRef(9.),
+	fTTLowSig(20.),
+	fTTUpSig(50.),
+	fNRPBins(50),
+	fFrac(0.8),
+	fJetEtaMin(-.5),
+	fJetEtaMax(.5),
+	fJetHadronDeltaPhi(0.6),
+	fJetContName(""),
+	fRandom(0),
+	fHistEvtSelection(0x0), 
+	fHJetSpec(0x0),
+	fh1TrigRef(0x0),
+	fh1TrigSig(0x0),
+	fh2Ntriggers(0x0),
+	fh2RPJetsC10(0x0),
+	fh2RPJetsC20(0x0),
+	fh2RPTC10(0x0),
+	fh2RPTC20(0x0)
+{
+  SetMakeGeneralHistograms(kTRUE);
+}
+
+/**
+ * Destructor
+ */
+AliAnalysisTaskJetCoreEmcal::~AliAnalysisTaskJetCoreEmcal()
+{
+}
+
+/**
+ * Performing run-independent initialization.
+ * Here the histograms should be instantiated.
+ */
+void AliAnalysisTaskJetCoreEmcal::UserCreateOutputObjects()
+{
+  AliAnalysisTaskEmcalJet::UserCreateOutputObjects();
+	cout<<"in task - create output objects"<<endl;
```

Comment:
Needs to be `std::cout` for ROOT 6 compatibility. Generally good to use AliDebugStream(int), AliInfoStream(), etc.

Reply:
Thanks, I didn't need this output so removed this line